src/panels/lovelace/common/create-badge-element.ts

```diff
@@ -0,0 +1,73 @@
+import deepClone from "deep-clone-simple";
+
+import "../badges/hui-entity-filter-badge";
+import "../badges/hui-state-label-badge";
+
+import {
+  createErrorBadgeElement,
+  createErrorBadgeConfig,
+  HuiErrorBadge,
+} from "../badges/hui-error-badge";
+import { LovelaceBadge } from "../types";
+import { LovelaceBadgeConfig } from "../../../data/lovelace";
+
+const BADGE_TYPES = new Set(["entity-filter", "error", "state-label"]);
+const CUSTOM_TYPE_PREFIX = "custom:";
+const TIMEOUT = 2000;
+
+const _createElement = (
+  tag: string,
+  config: LovelaceBadgeConfig
+): LovelaceBadge | HuiErrorBadge => {
+  const element = document.createElement(tag) as LovelaceBadge;
+  try {
+    element.setConfig(deepClone(config));
```

Comment:
Why are we deep cloning it ?

Reply:
Copied code from create-card. I didn't question it.

Reply:
😄 question everything. @bramkragten do you remember?